<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Automation-win" name="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest" tests="10" failures="1" timestamp="2023-12-14T13:45:31 IST" time="1214.941" errors="0">
  <testcase name="Annexure_Page_Test" time="4.223" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Creating_Sub_AuditTypes_Test" time="450.675" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Creating_Parent_Audit_Type_Test" time="291.986" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Uploading_Annexure_Documents_Test" time="38.172" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Auditee_Admin_Configuration_Test" time="17.225" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Creating_Audit_Schedule_Test" time="27.561" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Executing_The_Schedule_By_First_Auditor_Test" time="212.872" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Executing_The_Schedule_By_Second_Auditor_Test" time="100.921" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Finalize_Audit_Test" time="64.251" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest"/>
  <system-out/>
  <testcase name="Validating_The_Scores_Test" time="7.055" classname="scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest">
    <failure type="java.lang.AssertionError" message="Total Score is Wrong expected [58.5 / 110.0] but found [68.5 / 110.0]">
      <![CDATA[java.lang.AssertionError: Total Score is Wrong expected [58.5 / 110.0] but found [68.5 / 110.0]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest.Validating_The_Scores_Test(MultipleAuditorWithSubAuditLevelFunctionalityTest.java:4828)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validating_The_Scores_Test -->
  <system-out/>
</testsuite> <!-- scripts.MultipleAuditorWithSubAuditLevelFunctionalityTest -->
